### PR TITLE
Mike/post

### PR DESCRIPTION
**Pull request by michaelatimore**
Individual post. Post.tsx is populated using the imported props from PostData in content.ts

### CODE REVIEW

**Comment by michaelatimore** [comment]
Removed unused useState.

File: src/components/Post.tsx [hunk 1]
```diff
@@ -47,8 +47,7 @@ export function Post({ post }: PostProps) {
                 />
 
                 <div className="mx-4">
-                  <h1 className="text-sm text-gray-700 dark:text-gray-200">{post.author}</h1>
-                  <p className="text-sm text-gray-500 dark:text-gray-400">Lead Developer</p>
+                  <h1 className="text-sm text-gray-700 dark:text-gray-200">{post.author}</h1>
```
Comment (ChrisForti) [inline src/components/Post.tsx]: Now your getting the hang of it👍🏼

File: src/components/Post.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      {/* Container for the blog post */}
+      <section className="bg-white dark:bg-gray-900">
+        <div className="container px-6 py-10 mx-auto">
+          <h1 className="text-2xl font-semibold text-gray-800 capitalize lg:text-3xl dark:text-white">From the blog</h1>
+
+          <div className="mt-8 lg:-mx-6 lg:flex lg:items-center">
+            {/* Image of the blog post */}
+            <img
+              className="object-cover w-full lg:mx-6 lg:w-1/2 rounded-xl h-72 lg:h-96"
+              src={post.postImage}
```
Comment (ChrisForti) [inline src/components/Post.tsx:19]: I could not get this to render in my code.

**Review by ChrisForti — APPROVED** [review]
Shake and Bake👍🏼

**Pull request merged**